### PR TITLE
Update ftdi-vcp-driver to 2.3

### PR DESCRIPTION
**Pull request by xyb**

### CODE REVIEW

File: Casks/ftdi-vcp-driver.rb [hunk 1]
```diff
@@ -1,14 +1,20 @@
 cask :v1 => 'ftdi-vcp-driver' do
-  version '2.2.18'
-  sha256 '248f582527175ab07334f6a1255c8be2da4dfc36da49e26d39a2f61f5b7ba2e7'
+  if MacOS.release <= :mountain_lion
+    version '2.2.18'
+    sha256 '248f582527175ab07334f6a1255c8be2da4dfc36da49e26d39a2f61f5b7ba2e7'
+    pkg 'FTDIUSBSerialDriver_10_4_10_5_10_6_10_7.mpkg'
```
Comment (vitorgalvao) [inline Casks/ftdi-vcp-driver.rb]: The name here seems to suggest this works for `MacOS.release <= :lion`.

Comment (xyb) [inline Casks/ftdi-vcp-driver.rb]: I have double checked on [official site](http://www.ftdichip.com/Drivers/VCP.htm), which said the version `2.2.18` does work on `Mac OS X 10.3 to 10.8`. I guess it works just fine when 10.8 released, so they decide do not modify it anymore even the file name.

Comment (vitorgalvao) [inline Casks/ftdi-vcp-driver.rb]: Sure thing, just wanted to make sure it wasn’t a typo.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as b9857f47d7beb654892b3ea88e5f127a75f33746 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**